resources/views/fields/select.blade.php

```diff
@@ -3,6 +3,7 @@
         data-select-placeholder="{{$attributes['placeholder'] ?? ''}}"
         data-select-allow-empty="{{ $allowEmpty }}"
         data-select-message-notfound="{{ __('No results found') }}"
+        data-select-allow-add="{{ $allowAdd }}"
```

Comment:
If we're using `true/false` values then maybe we should use `var_export($allowAdd)` to make sure the values are always the same.